Sentry.xcodeproj/xcshareddata/xcschemes/Sentry.xcscheme

```diff
@@ -88,6 +88,9 @@
                <Test
                   Identifier = "SentryUIApplicationTests/test_applicationWithScenes_noWindow()">
                </Test>
+               <Test
+                  Identifier = "SentryViewHierarchyTests">
```

Comment:
`m`: Why do we need to skip this?

Reply:
Im still doing tests. Im not 100% confident that the changes applied here are reliable.

I'll let you know when I think we can merge this for a last review.